Runtime/Manager/BugSplatManager.cs

```diff
@@ -15,15 +16,30 @@ public sealed class BugSplatManager : MonoBehaviour
 
 		[SerializeField]
 		[Tooltip("Register BugSplat to capture LogType.Exceptions on initialization.")]
-		private bool registerLogMessageRecieved;
+		private bool registerLogMessageReceived = true;
 
-		private BugSplatManagerImpl _impl;
-		public BugSplat BugSplat => _impl.BugSplat;
+		private BugSplatRef bugsplatRef;
+		public BugSplat BugSplat => bugsplatRef.BugSplat;
```

Comment:
C# and Unity are a little funky. The BugSplat object is available soon after the program starts, but not immediately. I don't fully understand this, I'm kind of just following the pattern Zander established, but it has something to do with the way C# handles references, and by adding this layer of indirection we actually get the object we're looking for when it's time to call it. Moved the actual functionality into this class so we can use the MonoBehaviour.StartCoroutine method instead of passing around the game object.